### PR TITLE
Turn packet list into table

### PR DESCRIPTION
**Pull request by Arthri**
Turn packet list into table. i.e., this:
- Connect Request
- Disconnect

into

| ID  | Name |
|----|--------|
| 1  | Connect Request |
| 2 | Disconnect |

### CODE REVIEW

**Comment by hakusaro** [comment]
Did you test/write these in Obsidian and verify that changing the links from paths won't break them? I don't think they will, but just so we're on the same page, you did, right?

**Comment by hakusaro** [comment]
Also why is `N P C` spaced?

**Comment by Arthri** [comment]
> Also why is `N P C` spaced?

oops, my bad. I used regex to add spaces before caps

**Comment by Arthri** [comment]
> Did you test/write these in Obsidian and verify that changing the links from paths won't break them? I don't think they will, but just so we're on the same page, you did, right?

yeah, they worked

**Comment by hakusaro** [comment]
> oops, my bad. I used regex to add spaces before caps

bruh

File: docs/preset permission group mappings.md [hunk 1]
```diff
@@ -1,150 +1,150 @@
-|Permission|[[guest]]|[[default]]|[[vip]]|[[newadmin]]|[[admin]]|[[trustedadmin]]|[[owner]]|[[superadmin]]|
```
Comment (hakusaro) [inline docs/preset permission group mappings.md]: This file is auto generated by TShock. Please don't change it.

**Review by hakusaro — CHANGES_REQUESTED** [review]
There's at least one file that's autogenerated by TShock that's changed here, so please don't change that.

For other changes: I'll take a look in Obsidian tomorrow. I'm not opposed to the changes related to table spacing, but I feel like I need more justification as to why we should add a ton of whitespace around everything so that the table looks nice in fixed width fonts when it's rendered in non-fixed-width fonts. Markdown uses the 2nd line with the `|----|` stuff to determine the relative placement of the elements in the table. So adding space is unnecessary on other parts of the table, at least in my understanding of the language. This means that it's aesthetic changes that require more work to comply with in the long term while the docs are still being chiseled away at, imho. If you can elaborate more on why you decided to do this, I'd be less ambivalent I think.

Please try to avoid grab bag PRs like this that are size `XXL`. If you want to update the docs, please aim for size/M or lower unless you're performing a giant content update in one burst. You have multiple disparate changes in this including reformatting, content, headers, etc. This makes it extremely difficult to review. Yes, the bar is significantly lower for a wip branch, but it's still a diff that I have to read.

**Comment by hakusaro** [comment]
Also, there's no PR title

**Review by hakusaro — APPROVED** [review]

**Pull request merged**